tests/nested/manual/uc20-install-in-initrd/task.yaml

```diff
@@ -4,7 +4,7 @@ details: |
   This test checks that we are able to perform a single-boot installation in
   various scenarios.
 
-systems: [ubuntu-20.04-64]
+systems: [ubuntu-22.04-64, ubuntu-24.04-64]
```

Comment:
Why disable it for 20?

Reply:
+1 on this

Reply:
Whoops, not sure how that happened. Re-enabled.